api/tests/ticket_test.py

```diff
@@ -140,17 +139,18 @@ def setUp(self):
         self.ticket_data = {
             "assigned_id": self.assigned_user.id,
             "title": "Sic Mundus Creatus Est",
-            "ticket_number": 1,
             "team_id": self.team.id,
         }
         self.ticket_get_data = {
             "title": "Sic Mundus Creatus Est",
-            "ticket_number": 1,
             "team_id": self.team.id,
         }
-        self.response = self.ticket_client.post(
-            reverse("ticket-create-record"), self.ticket_data, format="json"
-        )
+
+        self.response = None
+        for i in range(1):
+            self.response = self.ticket_client.post(
+                reverse("ticket-create-record"), self.ticket_data, format="json"
+            )
```

Comment:
Why is this a for loop? for i in range(1) will just run once right?

Reply:
yes. during testing i changed the value around so i could see what happens when we create multiple tickets with the same data.

Reply:
Smart, besides that everything looks great. Merging now.